lib/metrix/supervisor.ex

```diff
@@ -29,4 +36,10 @@ defmodule ElixirTools.Metrix.Supervisor do
 
   @spec add_recurrent_metric_children([child]) :: [child]
   defp add_recurrent_metric_children(children), do: children ++ @recurrent_metrics
+
+  @spec metrix_enabled? :: boolean
+  defp metrix_enabled? do
+    enabled_values = [nil, true, "true"]
```

Comment:
nil == if no value is set
true == if it's "manually" set to true
"true" == if it's read from System env ?

Reply:
yes

Reply:
Is this gonna be clear to everyone in 2 weeks when they look at this?

Reply:
yes, because you were able to read it without context as well 😉

Reply:
I _assumed_, that's why I asked to be double sure 😇
But I cannot think of a more efficient way, so this is fine!